src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -33,26 +33,35 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public class RangeConstraint : Constraint
     {
-        private readonly IComparable from;
-        private readonly IComparable to;
+        private readonly object from;
+        private readonly object to;
 
         private ComparisonAdapter comparer = ComparisonAdapter.Default;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="RangeConstraint"/> class.
         /// </summary>
-        /// <remarks>from must be less than or equal to true</remarks> 
-        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
-        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
+        /// <param name="from">Must be less than or equal to the <paramref name="to"/> value.</param>
+        /// <param name="to">Must be greater than or equal to the <paramref name="from"/> value.</param>
         public RangeConstraint(IComparable from, IComparable to) : base( from, to )
```

Comment:
This must be (object, object). An `Is.InRange(notIComparable, notIComparable).Using(customComparer)` test would force this to be done. /cc @rprouse